### PR TITLE
Added Error Handlebar

### PR DESCRIPTION
**Pull request by galapatt**
-  added a new error handlebar
- more css for the error handlebar
- added helper functions to handlebar

### CODE REVIEW

File: routes/users.js [hunk 1]
```diff
@@ -4,11 +4,16 @@ import {userData} from '../data/index.js';
...
 
 router
-  .route('/')
-  
-
-router
-  .route('/:id')
+  .route('/:id').get(async (req,res) => {
+    try {
+      var userId = validation.checkValidId(req.params.id, "User ID")
```
Comment (vishwa5854) [inline routes/users.js]: I think u have forgotten about the response in success case where the id is valid
and is this a testing route ??

File: app.js [hunk 2]
```diff
@@ -39,6 +39,19 @@ app.use(rewriteUnsupportedBrowserMethods);
...
+// register new function
+hbs.handlebars.registerHelper({
+  eq: (v1, v2) => v1 === v2,
+  ne: (v1, v2) => v1 !== v2,
+  lt: (v1, v2) => v1 < v2,
+  gt: (v1, v2) => v1 > v2,
+  lte: (v1, v2) => v1 <= v2,
+  gte: (v1, v2) => v1 >= v2,
+  capitalize : (str) => str.charAt(0).toUpperCase() + str.slice(1)
+})
```
Comment (vishwa5854) [inline app.js:53]: May I know the significance of these functions ?? I haven't used them yet or seen before

Comment (galapatt) [reply app.js:53]: I use it in the error handlebar to check if the value is equal to a specific errorCode. They are helper functions to use in your handlebars for the conditional HTML.

File: routes/users.js (hunk 1 above)
Comment (galapatt) [reply routes/users.js]: I am just testing the error handlebars for now. I haven't implemented the actual html page yet.

**Review by vishwa5854 — APPROVED** [review]

**Pull request merged**